### ISSUE TITLE
[Accessibility issue]Unable to expand dropdown options and unable select dropdown values using keyboard(Tab, Up/Down arraows)

### ISSUE THREAD

**Issue by DGanapathi** [manishjanky/ngx-select-dropdown#56]
Repro steps:

1. Focus dropdown using tab key
2. Click on down arrow to expand the list**(Not working in any browser)**
3. Expand the list manually and Try to type anything to get filtered list**(Not able to type in any browser)** Need to click on search textbox using mouse to type search text
4. Try to navigate between drop down options using up&down arrows from keyboard**(Not working in IE11 & Edge)**

**Comment by manishjanky** [comment]
@DGanapathi `tabindex` support is not there. Will add that soon and update you.

**Comment by DGanapathi** [comment]
Thank you Manish.
Also please consider 4th point (Navigation between drop down options using **up & down** arrows from keyboard Not working in Edge &  IE11)

**Comment by DGanapathi** [comment]
Any update on this?

**Comment by manishjanky** [comment]
Hi 
Please update to the latest version. This has been implemented. Please let me know if you face any issues.

**Comment by DGanapathi** [comment]
Fixed partial.
Not working below scenarios
1. After expanding the dropwon list by pressing down arrow, search box is not auto focused. Need to hit Tab two times to get focus.
2. Navigation between drop down options using up&down arrows from keyboard Not working in IE11 & Edge
3. Dropdown view is not getting closed on Escape key press or focus out using tab key.

**Comment by DGanapathi** [comment]
Could you please reopen this bug as it is not working above scenarios in IE11 & Edge?

**Comment by Abhishek-Kanitkar** [comment]
> Fixed partial.
> Not working below scenarios
> 
> 1. After expanding the dropwon list by pressing down arrow, search box is not auto focused. Need to hit Tab two times to get focus.
> 2. Navigation between drop down options using up&down arrows from keyboard Not working in IE11 & Edge
> 3. Dropdown view is not getting closed on Escape key press or focus out using tab key.

@manishjanky  can you please reopen this defect ? I also need it in my project.
